test/test_file_blob.rb

```diff
@@ -188,6 +188,10 @@ def test_generated
     assert fixture_blob("Binary/MainMenu.nib").generated?
     assert !sample_blob("XML/project.pbxproj").generated?
 
+    # Cocoapods
+    assert sample_blob('Pods/blah').generated?
+    assert !sample_blob('My-Pods/blah').generated?
```

Comment:
@lildude It's a bit more straightforward in this test file to add the "negative" test, so I did. I think this is OK for now to test the regex validity but up to you :)